### PR TITLE
Implement workspace module

### LINKED ISSUE #3673: OGIP 17: Package onegovgever.workspace mit Feature-Flag erstellen
- Neues Sub-Package `onegovgever.workspace` in `opengever.core` erstellen
- Neues Feature-Flag für das Workspace-Package erstellen; Feature-Flag standardmässig deaktivert, für Development aktiviert

### PR DESCRIPTION
**Pull request by raphael-s**
Adds the workspace module suggested in OGIP 17. It doesn't do anything yet, its just an empty module with a feature flag set to false by default.

closes #3673

### CODE REVIEW

File: opengever/workspace/configure.zcml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    xmlns:five="http://namespaces.zope.org/five"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    i18n_domain="opengever.workspace">
+
+  <!-- Register an extension profile to make the product installable -->
+  <genericsetup:registerProfile
```
Comment (phgross) [inline opengever/workspace/configure.zcml]: @jone is it really the plan to introduce a separate profile for the workspace stuff?

Comment (jone) [reply opengever/workspace/configure.zcml]: No, it is not.
@raphael-s in GEVER we now only use the profile at `opengever/core/profiles/default` (`opengever.core:default`). Please put everything there and remove this profile.

**Comment by jone** [comment]
And please add a changelog entry.

**Comment by raphael-s** [comment]
@jone I updated the translation files, removed the empty workspace profile and added a changelog entry.

File: opengever/workspace/configure.zcml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:five="http://namespaces.zope.org/five"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    i18n_domain="opengever.workspace">
+
+  <i18n:registerTranslations directory="locales" />
+
+  <include package=".upgrades" />
```
Comment (jone) [inline opengever/workspace/configure.zcml]: ✂️

Comment (raphael-s) [reply opengever/workspace/configure.zcml]: Already removed it 😄

**Pull request merged**